mart/callbacks/anomalib.py

```diff
@@ -0,0 +1,425 @@
+#
+# Copyright (C) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+from __future__ import annotations
+
+import os
+import types
+from copy import deepcopy
+
+import torch
+from lightning.pytorch.callbacks import Callback
+
+from torch_rotation import rotate_three_pass
```

Comment:
This is a requirement that should be added or somehow specified somewhere.
`pip install torch-rotation==0.1.2`

Reply:
Fixed in 2c8c2768f4b76c4b4d94f9330bc90b7783839af6.